### PR TITLE
Algorithm 001_maxOfThree  - JS Solution

### PR DESCRIPTION
**Pull request by satyasaadhak**
@abhi18av I have implemented a JS solution for this algorithm that takes the arguments as an input from the user, and outputs the result.

Do review the code and the result.

### CODE REVIEW

**Comment by abhi18av** [comment]
@satyasaadhak , I appreciate you taking the time and energy to work on this PR but I must tell you that for now we are only sticking to the base libraries and no external packages at all. 

I'm going to close this PR and I request you to raise another PR - you could simplify the case by reading the numbers from a json file, perhaps called `input.json` within the directory.

Sorry :)

File: 001_maxOfThree/maxOfThree.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const argv = require('yargs')
```
Comment (abhi18av) [inline 001_maxOfThree/maxOfThree.js]: Let's rely only on the native libs and master the usage of what comes right inside the box.

File: 001_maxOfThree/maxOfThree.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              .usage('Usage: $0 --x=[num] --y=[num] --z=[num]')
+              .demand(['x', 'y', 'z'])
+              .argv;
+
+/**
```
Comment (abhi18av) [inline 001_maxOfThree/maxOfThree.js]: Happy to note that you've paid some attention to the documentation.

**Comment by satyasaadhak** [comment]
> @satyasaadhak , I appreciate you taking the time and energy to work on this PR but I must tell you that for now we are only sticking to the base libraries and no external packages at all.
> 
> I'm going to close this PR and I request you to raise another PR - you could simplify the case by reading the numbers from a json file, perhaps called `input.json` within the directory.
> 
> Sorry :)

Sure man, will code a new version today. My intention was to allow user input to the application. Let's see if there's any native solution to this, or I'll just use hard coded inputs or input files.

File: 001_maxOfThree/maxOfThree.js (hunk 2 above)
Comment (satyasaadhak) [reply 001_maxOfThree/maxOfThree.js]: I feel it's important that it's clear how to use the application to the user, and the code is maintainable for other developers who work on my code.

**Comment by abhi18av** [comment]
The latter option works better for now - later on we could move to mini `cli tools creation` challenges :)

**Pull request closed without merging**